repos/system_upgrade/el7toel8/libraries/tests/test_isccfg.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env python
+#
+# Tests for bind configuration parsing
+
+try:
+    from leapp.libraries.common import isccfg
+except ImportError:
+    # For easier testing, allow having it out-of-directory
+    import isccfg
```

Comment:
I don't understand it. Do you modify a system path?

Reply:
No. This allowed me testing the same file out of  repos structure by *pytest-3 test_isccfg* on Fedora. Because I had a lot of code to cleanup, debugging via Makefile was quite painful to me. I needed debugger ocassionally. This was easy workaround.

I don't modify path. But any python file in leapp repos needs running under make for some stupid reason. This allowed me to use pytest directly in different directory. I would like better fix, but this just worked enough.

Reply:
@pemensik This is already improved by @ZhukovGreen . You can run pytest nowadays without the makefile: https://leapp.readthedocs.io/en/latest/unit-testing.html#actor-s-tests

Reply:
Yeah, this should be fixed. Sorry you had the user-unfriendly experiences with our tests, it should be better now.